src/functions/module_statistical.ts

```diff
@@ -599,6 +909,138 @@ export const LARGE = {
   isExported: true,
 } satisfies AddFunctionDescription;
 
+// -----------------------------------------------------------------------------
+// LINEST
+// -----------------------------------------------------------------------------
+export const LINEST: AddFunctionDescription = {
+  description: "Compute the intercept of the linear regression.",
+  args: [
+    arg(
+      "data_y (range<number>)",
+      _t("The range representing the array or matrix of dependent data.")
+    ),
+    arg(
+      "data_x (range<number>, optional, default = {1,2,3,...})",
+      _t("The range representing the array or matrix of independent data.")
+    ),
+    arg(
+      "calculate_b (boolean, default=TRUE)",
+      _t("A flag specifying wheter to compute the slope or not")
+    ),
+    arg(
+      "verbose (boolean, default=FALSE)",
+      _t(
+        "A flag specifying whether to return additional regression statistics or only the linear coefficients and the y-intercept"
+      )
+    ),
+  ],
+  returns: ["NUMBER"],
+  compute: function (
+    dataY: Matrix<number>,
+    dataX: Matrix<number> = [],
+    calculate_b: boolean = true,
+    verbose: boolean = false
+  ): CellValue[][] {
+    return fullLinearRegression(dataX, dataY, calculate_b, verbose);
+  },
+  isExported: true,
+};
+
+// -----------------------------------------------------------------------------
+// LOGEST
+// -----------------------------------------------------------------------------
+export const LOGEST: AddFunctionDescription = {
+  description: "Compute the intercept of the linear regression.",
+  args: [
+    arg(
+      "data_y (range<number>)",
+      _t("The range representing the array or matrix of dependent data.")
+    ),
+    arg(
+      "data_x (range<number>, optional, default = {1,2,3,...})",
+      _t("The range representing the array or matrix of independent data.")
+    ),
+    arg(
+      "calculate_b (boolean, default=TRUE)",
+      _t("A flag specifying wheter to compute the slope or not")
+    ),
+    arg(
+      "verbose (boolean, default=FALSE)",
+      _t(
+        "A flag specifying whether to return additional regression statistics or only the linear coefficients and the y-intercept"
+      )
+    ),
+  ],
+  returns: ["NUMBER"],
+  compute: function (
+    dataY: Matrix<number>,
+    dataX: Matrix<number> = [],
+    calculate_b: boolean = true,
+    verbose: boolean = false
+  ): CellValue[][] {
+    const coeffs = fullLinearRegression(dataX, logM(dataY), calculate_b, verbose);
```

Comment:
with dataX and dataY as ```Matrix<CellValue>```, sure there will never be anything than numbers in coeffs?